docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "43cd9359-32f3-455a-b9a2-3f419adacc32",
+   "metadata": {},
+   "source": [
+    "July 2024 was an exceptional month in terms of irradiance anomalies in North America. In one of [Solcast's weekly PVMag publications](https://www.pv-magazine.com/2024/08/02/aerosols-from-wildfires-slash-north-american-solar-production/) we have highlighted how irradiance was severely impacted in the north west of  the continent by aerosols deriving from the raging Canadian wildfires.\n",
+    "\n",
+    "In addition to the reduced irradiance from the aerosol's absorption, another factor to take into account into the overall impact of these aerosols on solar generation is dust soiling as the accumulated particles block or scatter incident light, further reducing power output.\n",
```

Comment:
The example doesn't actually show any aerosol-based modelling or make any further mention of aerosols.  In one of the cases you show subsequently, should you at least do something like say "if there is more soiling than usual, e.g. due to the increased aerosols in the atmosphere with wildfires, we can increase the soiling_loss_rate to represent this", and then show this in the example?

If keeping the mention of aerosols in this introduction, this sentence should be reworded as it is a bit awkward.  Better would be "In addition to the reduced irradiance from the aerosol's absorption, the overall impact of these aerosols on solar generation can include dust soiling..."